### PR TITLE
DOCS-2892: Reduce board confusion

### PR DESCRIPTION
**Pull request by JessamyT**

### CODE REVIEW

File: docs/installation/_index.md [hunk 1]
```diff
@@ -74,9 +74,7 @@ Proceed to [install `viam-micro-server`](/installation/#install-viam-micro-serve
 
 ### Operating System installation
 
-`viam-server` can run on **any computer that runs a [supported operating system](#platform-requirements)**, including single-board computers (SBCs) running 64-bit Linux.
```
Comment (npentrel) [inline docs/installation/_index.md:77]: I think that's important to restate.

**Comment by npentrel** [comment]
One comment otherwise good

File: docs/installation/_index.md [hunk 2]
```diff
@@ -74,9 +74,7 @@ Proceed to [install `viam-micro-server`](/installation/#install-viam-micro-serve
...
-
-For convenience, we provide operating system installation instructions for some popular SBCs.
+For convenience, we provide operating system installation instructions for some popular single-board computeres (SBCs).
```
Comment (JessamyT) [inline docs/installation/_index.md:77]:
```suggestion
`viam-server` can run on **any computer that runs a [supported operating system](#platform-requirements)**, including single-board computers (SBCs) running 64-bit Linux.

For convenience, we provide operating system installation instructions for some popular single-board computeres (SBCs).
```

File: docs/installation/_index.md [hunk 3]
```diff
@@ -76,7 +76,7 @@ Proceed to [install `viam-micro-server`](/installation/#install-viam-micro-serve
 
 `viam-server` can run on **any computer that runs a [supported operating system](#platform-requirements)**, including single-board computers (SBCs) running 64-bit Linux.
 
-For convenience, we provide operating system installation instructions for some popular SBCs.
+For convenience, we provide operating system installation instructions for some popular single-board computeres (SBCs).
```
Comment (JessamyT) [inline docs/installation/_index.md:79]:
```suggestion
For convenience, we provide operating system installation instructions for some popular SBCs.
```

**Review by npentrel — APPROVED** [review]

**Pull request merged**